projects/plugins/boost/app/modules/optimizations/lazy-images/class-lazy-images.php

```diff
@@ -16,6 +16,13 @@ public static function get_slug() {
 	}
 
 	public static function is_available() {
+		// In some scenarios, Lazy Images can conflict with other WordPress features.
+		// If we detect that, the feature should not be available.
+		if ( Jetpack_Lazy_Images::should_force_deactivate() ) {
+			Jetpack_Lazy_Images::disable();
```

Comment:
I would recommend that you avoid disabling the module here. This function is for checking the availability and disabling the module would be a side effect. Besides, it would disable the module in the Jetpack plugin.

In practice, returning false from this function if force deactivation is needed should be enough for it to work.

Reply:
👍 Removed that in 72fc3ec3779fe0478ca373629a02a481a598e4c0